package.json

```diff
@@ -33,5 +33,10 @@
     "prepublishOnly": "npm run build && cp README.md packages/core/README.md && cp README.md packages/react/README.md",
     "postpublish": "rm -rf packages/core/README.md && rm -rf packages/react/README.md",
     "clean": "lerna run --stream clean"
+  },
```

Comment:
Should be in the `react` package's `package.json`, not here. Or ideally in `mantine`, `shadcn`, and `ariakit`, once you extract the emoji components to those.